### DISCUSSION TITLE
To understand the connection cable between Gateway E32 V2 and Bosch Compress WP

### DISCUSSION THREAD

**Discussion by leika65** [emsesp/EMS-ESP32#1629] [Q&A]
Hello,
I have a general question about wiring because I'm completely new to this area.
I ordered the “Gateway E32 V2 (Ethernet + WiFi Edition V2)” module to connect my Bosch heat pump via the EMS port.
I'm not sure whether I ordered the correct connection cables. I also ordered the cable “EMS service cable 110cm” and the “12V DC power supply (2 pin Euro plug)”.
My heat pump is the Bosch Compress 5800i with the Compress 6800iAW 12M indoor unit.
I also have the UI800 CS control panel and the CR20 RF remote control.
I hope I ordered the correct cable.
Is there actually a precise connection description of how to connect the gateway to the Compress 6800iAW via EMS?
Thanks for your help

**Comment by proddy** [comment]
that's a @bbqkees question !

**Comment by bbqkees** [comment]
It's all explained in detail in the wiki:
https://bbqkees-electronics.nl/wiki/gateway/connecting-to-the-boiler.html#service-jack-on-heat-pumps

Under the hood of the indoor unit the service jack is on the end of a small cable that is connected to the BBT screw terminal.
Here you can connect the E32 V2 with the purchased EMS service cable 110cm.
You do not need to connect the 12V DC power supply, as the service jack already provides the 12V to the EMS Gateway.
However you can still connect the 12V DC power supply so that in case of a hard reboot of the heat pump etc the EMS Gateway won't lose power.
